Userland/Libraries/LibJS/Contrib/Test262/IsHTMLDDA.h

```diff
@@ -18,10 +18,9 @@ class IsHTMLDDA final : public NativeFunction {
     virtual ~IsHTMLDDA() override = default;
 
     virtual ThrowCompletionOr<Value> call() override;
-    virtual ThrowCompletionOr<Object*> construct(FunctionObject& new_target) override;
 
 private:
-    virtual bool has_constructor() const override { return true; }
+    virtual bool has_constructor() const override { return false; }
```

Comment:
[No need to override this either](https://github.com/SerenityOS/serenity/blob/78bc84c2d821d2f5e7bd9808b011fe94173853e9/Userland/Libraries/LibJS/Runtime/FunctionObject.h)

Reply:
Yeah I left that to make it explicit it shouldn't be constructible but this is cleaner.